src/main/java/org/myrobotlab/opencv/OpenCVFilterYolo.java

```diff
@@ -90,8 +79,8 @@ public OpenCVFilterYolo() {
     this(null);
   }
 
-  private void downloadYoloModel() {
-    log.info("downloadYoloModel - begin");
+  private void checkFiles() {
+    log.info("checkFiles - begin");
     File yoloHome = new File(darknetHome);
     if (!yoloHome.exists()) {

```

Comment:
this method isn't downloading files anymore.. it shouldn't create the files.  additionally, this method doesn't return and value, so the caller of this method doesn't know if the files were valid.  This method should either be removed, or it should include a return value that has some useful meaning.

Reply:
will remove it